ResearchKit/Consent/ORKEAGLMoviePlayerView.m

```diff
@@ -246,10 +258,57 @@ - (void)cleanUpTextures {
     
     // Periodic texture cache flush every frame
     CVOpenGLESTextureCacheFlush(_videoTextureCache, 0);
+    
+    [self restoreGLContext];
+}
+
+- (BOOL)saveGLContext {
```

Comment:
I'm not sure I like "save" as the name here. "Save" normally means you don't change anything - but here "save" has the side effect of setting the current context.

Could we instead call these push / pop? e.g. pushGLContext / popGLContext